pkg/types/nutanix/platform.go

```diff
@@ -40,8 +39,37 @@ type Platform struct {
 	// +optional
 	DefaultMachinePlatform *MachinePool `json:"defaultMachinePlatform,omitempty"`
 
-	// SubnetUUID specifies the UUID of the subnet to be used by the cluster.
-	//
+	// SubnetUUID identifies the network subnet to be used by the cluster.
 	// +optional
-	SubnetUUID string `json:"subnetUUID,omitempty"`
+	SubnetUUID string `json:"subnetUuid,omitempty"`
+}
+
+// NutanixPrismCentral holds the endpoint and credentials data used to connect to the Prism Central
+type NutanixPrismCentral struct {
+	// Endpoint holds the address and port of the Prism Central
+	Endpoint configv1.NutanixPrismEndpoint `json:"endpoint"`
+
+	// Username is the name of the user to connect to the Prism Central
+	Username string `json:"username"`
+
+	// Password is the password for the user to connect to the Prism Central
+	Password string `json:"password"`
+}
+
+//
+type NutanixPrismElement struct {
+	// name is the name of the Prism Element (cluster)
+	Name string `json:"name"`
+
+	// uuid
+	UUID string `json:"uuid"`
```

Comment:
Can we deduce the UUID from the name, or vice versa? If not, then we need validation that the user-provided name and UUID correspond to the same Prism Element.